tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -1281,6 +1281,39 @@ Status Converter::GetInputs(const tensorflow::NodeDef& node_def,
   return tensorflow::Status::OK();
 }
 
+// Checks that the number of inputs match, and enforces that the inputs marked
+// as true are constant weights. true means that the input must be a weight,
+// while false means the input must be a tensor. In the future, false will mean
+// the input can be a tensor or weight.
+tensorflow::Status CheckInputsWeights(
```

Comment:
Take both params by const reference so that it is clear that they are inputs. And in case of params, that it can't be nullptr.